### PR TITLE
Add php-cs-fixer as composer script and make initial fix-style run

### PR DESCRIPTION
**Pull request by jasonrhodes**
Let's get a few eyes on this to make sure it looks ok, but this will make style stuff way easier imo.

### CODE REVIEW

**Comment by richleland** [comment]
@jasonrhodes has anyone reviewed this yet?

**Comment by beardyman** [comment]
@richleland - nobody has.  I can.

File: lib/SparkPost/APIResource.php [hunk 1]
```diff
@@ -174,29 +199,32 @@ private function buildBody($options) {
...
    */
-  private function callResource( $action, $resourcePath=null, $options=[] ) {
-    $action = strtoupper($action); // normalize
+  private function callResource($action, $resourcePath = null, $options = [])
+  {
+      $action = strtoupper($action); // normalize
 
     $url = $this->buildUrl($resourcePath, $options);
-    $body = $this->buildBody($options);
+      $body = $this->buildBody($options);
```
Comment (beardyman) [inline lib/SparkPost/APIResource.php]: I like most of what has been done here but I don't think that it handled the spaces to tabs thing very well.  These 3 lines end up having `$action` and `$body` both indented farther than `$url`

**Comment by beardyman** [comment]
So, I'm noticing that spacing issue a lot.  The PSR-2 standard says:

> Code MUST use an indent of 4 spaces, and MUST NOT use tabs for indenting.

Looks like we're going to have to fix that one by hand.

**Comment by richleland** [comment]
What's next? Want me to go through and fix the space/tabs thing?

**Comment by beardyman** [comment]
Sure, I was thinking about scriptifying it so that this cs-fixer and that script could be ran easily.  Sort of like  our linting stuff for other projects.  Just haven't had time to do it.

**Pull request merged**